FBSnapshotTestCaseDemo/Podfile.lock

```diff
@@ -1,20 +0,0 @@
-PODS:
```

Comment:
I don't think the `Podfile.lock` should be deleted

Reply:
@alanzeino

Reply:
yeah it's used in the demo because the demo is a real app depending on FBSnapshotTestCase, therefore it needs a Podfile and a Podfile.lock